docs/src/phyto_equations.md

```diff
@@ -149,42 +149,100 @@ chl &= chl + S_{chl} \cdot \Delta T \nonumber
 
 ## Cell division
 
-Cell size, ``Sz``, is used to indicate cell division. For the smallest cell, ``Sz=1.0`` and ``Bm=Cquota``. Cells start to divide at ``Sz=2.0`` and the probability of individual cell division is then given by a sigmoidal function of ``Sz``.
+Relative cell size, ``Sz``, is used to indicate cell division. Cells will not divide if ``Sz<2.0`` and the probability of individual cell division (``P_D``) is then given by a sigmoidal function of ``Sz`` with different cell division strategies.
 
 ```math
 Sz = (Bm + Cq) / Cquota
 ```
 
 ```math
-P_{divide} = rand(Bernoulli(P_{dvid}*(tanh(stp_{dvid}*(Sz-b))+1)))
+P_D = rand(Bernoulli(P_{dvid}*S_{dvid}))
 ```
 
-``P_{divide}`` is computed every hour no matter what time step the model is.
+where ``S_{dvid}`` represents the sigmoidal function in different cell division strategies (listed below) and ``P_D`` is computed every 10 min (when model time step is smaller than 10 min) or every time step (when model time step is larger than 10 min).
+
+### Sizer
+
+The probability of cell division is a function of cell size and increase while the cell size gets larger.
```

Comment:
The probability of cell division is a function of cell size and increases as the cell size gets larger.